File: atcoder.jp/contests/abc205/tasks/abc205_b/main.py
from typing import Union


def yes_no(b: Union[bool, int]) -> str:
    return "Yes" if b else "No"


N = int(input())
A = [int(x) - 1 for x in input().split()]

bucket = [False] * N

for a in A:
    if bucket[a]:
        ans = False
        break
    bucket[a] = True
else:
    ans = True

print(yes_no(ans))

File: atcoder.jp/contests/abc205/tasks/abc205_b/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];
const print = (...messages: any) => console.log(...messages);

const yesNo = (b: boolean | number) => (b ? "Yes" : "No");

const N = +input();
const A = input()
  .split(" ")
  .map((x) => +x - 1);

const bucket: boolean[] = Array(N).fill(false);

let ans = true;
for (const a of A) {
  if (bucket[a]) {
    ans = false;
    break;
  }
  bucket[a] = true;
}

print(yesNo(ans));
